src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/model/CaseData.java

```diff
@@ -195,7 +195,7 @@ public class CaseData {
     @CCD(access = {DefaultAccess.class,SystemUpdateAccess.class})
     private SocialWorker applicantSocialWorker = new SocialWorker();
 
-    @JsonUnwrapped
+    @JsonUnwrapped(prefix = "solicitor")
```

Comment:
better to replace prefix as applicant. so that it will represented as  applicant solicitor. which is much better naming convension